index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title> Germany</title>
+</head>
+<body>
+<nav>
+  <ul>
+    <li> <a href="culture.html"> Culture </a> </li> 
+    <li> <a href="language.html"> Language </a> </li> 
+    <li> <a href="places.html"> Places </a> </li> 
+    <li> <a href="politics.html"> Politics </a> </li> 
+    <li> <a href="cbyx.html"> CBYX </a> </li>
+  </ul>
+</nav>
+  <h1> Wilkommen </h1>
+
+  <h2> Germany </h2>
+    <img src="images/germany.jpg" alt="Linderhof Palace">
+    <figcaption> 
+      <em> Linderhof Palace (2015) </em>
+    </figcaption>
+
+  <h3> Quick Facts about Germany</h3>
+    <ul>
+      <li> Germany has 16 states</li>
+      <br>
```

Comment:
Once you integrate with CSS be sure to remove the <br> and replace with appropriate styling as it is an html element that performs styling.